platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/test/java/com/swirlds/platform/test/sync/SyncNode.java

```diff
@@ -282,6 +301,14 @@ public Shadowgraph getShadowGraph() {
         return shadowGraph;
     }
 
+    /**
+     * Sets the current {@link NonAncientEventWindow} for the {@link Shadowgraph}.
+     */
+    public void updateEventWindow(@NonNull final NonAncientEventWindow eventWindow) {
+        System.out.println(eventWindow);
```

Comment:
I know we're in test code, but was it intended to leave this System out here?  Any other helpful text to indicate the understanding of why we are wanting to dump the event window to the console here?

Reply:
Artifact of me debugging a test, was not my intention to leave this here.